android/expoview/src/main/java/versioned/host/exp/exponent/ExpoTurboPackage.kt

```diff
@@ -88,6 +111,23 @@ class ExpoTurboPackage(
     }
   }
 
+  private fun createReanimatedUIManager(reactContext: ReactApplicationContext): UIManagerModule? {
+    ReactMarker.logMarker(CREATE_UI_MANAGER_MODULE_START)
+    Systrace.beginSection(Systrace.TRACE_TAG_REACT_JAVA_BRIDGE, "createUIManagerModule")
+    val reactInstanceManager = kernel.reactInstanceManager!!
+    val minTimeLeftInFrameForNonBatchedOperationMs = -1
+    return try {
+      ReanimatedUIManager(
+        reactContext,
+        reactInstanceManager.getOrCreateViewManagers(reactContext),
+        minTimeLeftInFrameForNonBatchedOperationMs
+      )
+    } finally {
+      Systrace.endSection(Systrace.TRACE_TAG_REACT_JAVA_BRIDGE)
+      ReactMarker.logMarker(CREATE_UI_MANAGER_MODULE_END)
+    }
```

Comment:
Let's add some comment here. Especially I'm curious about `try {} finally {}` blocks.
In what scenarios `try` would fail? 🤔

Reply:
to be honest, i don't know 😅  i didn't take much look inside the reanimated implementation. the code is referenced from here:

https://github.com/software-mansion/react-native-reanimated/blob/d0faff2f9dd3f09720e4e4735e81c5b88d533007/android/src/main/java/com/swmansion/reanimated/ReanimatedPackage.java#L74-L82